### PR TITLE
Add `kubernetes resources` support

### LINKED ISSUE #225: Support Kubernetes resource types in `MeshSync`
**Current Behavior**

Currently, MeshSync supports a few resource types:
- [x] Volume
- [x] Namespaces 
- [x] ConfigMaps
- [x] Nodes
- [x] Secrets
- [x] Persistent Volumes #120 
- [x] Persistent Volume Claims #120 
- [x] Replicasets 
- [x] Pods
- [x] Services
- [x] Deployments
- [x] StatefulSets
- [x] Daemonsets
- [x] Ingresses #219 
- [x] Endpoints #224 

**Desired Behavior**

There are many that we need to add support for:
- [x] EndPointSlice
- [x] CronJob
- [x] ReplicationController
- [x] APIService
- [x] StorageClass
- [x] VolumeAttachment
- [ ] Event
- [ ] EventSeries
- [ ] Binding
- [ ] ClusterRole
- [ ] ComponentStatus
- [ ] ResourceQuota
- [ ] Role
- [ ] IngressClass

And there are many more to add, will update this list as we go on adding.

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). 
- Fill-in a [Community Member Form](https://layer5.io/newcomer) and join the [Community Slack](http://slack.meshery.io) for access.

### PR DESCRIPTION
**Pull request by Aritra8438**
**Description**

This PR partially fixes #225.

**Notes for Reviewers**

In this pr, I will add more kubernetes resource types.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aritra8438** [comment]
// @leecalcote.

File: internal/config/default_config.go [hunk 1]
```diff
@@ -76,6 +76,36 @@ var (
...
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added ClusterRole support
+			{
+				Name:      "clusterroles.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added VolumeAttachment support
+			{
+				Name:      "volumeattachments.v1.",
```
Comment (leecalcote) [inline internal/config/default_config.go:106]: Quick check - @Aritra8438 all of these end in .v1. with no further qualification, right?

Comment (Aritra8438) [reply internal/config/default_config.go:106]: Hi @leecalcote, are you referring to `volumeattachments.v1.storage.k8s.io`

Considering this [review](https://github.com/meshery/meshsync/pull/219#discussion_r1297138159), I submitted this pr.

Comment (leecalcote) [reply internal/config/default_config.go:106]: @Aritra8438 I retract my prior guidance on that PR. I believe that is NOT the way to go, but that we need to fully match the API resource name instead.

Comment (Aritra8438) [reply internal/config/default_config.go:106]: No problem, @leecalcote.

**Comment by Aritra8438** [comment]
// @leecalcote

**Review by leecalcote — APPROVED** [review]
Looks good

**Comment by leecalcote** [comment]
@Aritra8438, you're in a position now to help get these resources exposed in MeshMap Visualizer.

**Pull request merged**